src/components/views/create/CreateProject.tsx

```diff
@@ -316,13 +317,23 @@ const CreateProject: FC<ICreateProjectProps> = ({ project }) => {
 								id: 'label.you_can_set_a_custom_ethereum_address',
 							})}
 						</CaptionContainer>
-						{networksIds.map(networkId => (
+						{ALL_CHAINS.map(chain => (
 							<AddressInterface
-								key={networkId}
-								networkId={networkId}
-								onButtonClick={() =>
-									setAddressModalChainId(networkId)
+								key={chain.id}
+								networkId={chain.id}
+								chainType={
+									'chainType' in chain
+										? chain.chainType
+										: undefined
 								}
+								onButtonClick={() => {
+									if ('chainType' in chain) {
+										setAddressModalChainType(
+											chain.chainType,
+										);
+									}
```

Comment:
```suggestion
									setAddressModalChainType(
										(chain as NonEVMChain).chainType,
									);
```

I guess the value must be updated on each change, otherwise, it will hold old value after button click